spec/lib/reporting/identity_verification_report_spec.rb

```diff
@@ -115,6 +117,7 @@
         'IdV Reject: Doc Auth' => 3,
         'IdV Reject: Phone Finder' => 1,
         'IdV Reject: Verify' => 1,
+        'Fraud: Profile review passed' => 1,
```

Comment:
this is an "event", like a literal CW event name right? so we put it up above?

Reply:
Thanks for the guidance. I was trying to figure out the patterns here, and adding success = 1 to the query is what I was missing.

Reply:
I changed the code to print out the query and then tested it out in cloudwatch. Fixed one problem, thanks for the suggestion.